.travis.yml

```diff
@@ -18,7 +18,8 @@ before_install:
     - phpenv config-rm xdebug.ini || return 0
 
 install:
-    - travis_retry composer install
+    - composer --prefer-source install
+    - if [[ $TRAVIS_PHP_VERSION = nightly ]]; then phpenv global 7.1; ./vendor/bin/simple-phpunit install; phpenv global nightly; fi
```

Comment:
why 7.1 ? Wouldn't using 7.4 be better ? A PHPUnit version compatible with PHP 7.4 is more likely to work on PHP 8 than one selected for PHP 7.1.